static-files/ocp_on_premise.yaml

```diff
@@ -0,0 +1,178 @@
+---
+generators:
+  - OCPGenerator:
+      start_date: 2019-05-01
+      end_date: 2019-07-31
+      nodes:
+        - node:
+          node_name: compute_1
+          cpu_cores: 4
+          memory_gig: 16
+          namespaces:
+            Toronto:
+              pods:
+                - pod:
+                  pod_name: pod_name1
+                  cpu_request: 1
+                  mem_request_gig: 2
+                  cpu_limit: 1
+                  mem_limit_gig: 4
+                  pod_seconds: 3600
+                  labels: label_application:Cost|label_stage:Production|label_group:Marketing
```

Comment:
Labels are controlled here. The `label_` is to mirror how labels are returned by Prometheus. It needs to be here in this file, but you won't see it in the UI, you would just see application, stage, group, et cetera.